rosbag2_py/src/rosbag2_py/format_bag_metadata.cpp

```diff
@@ -115,7 +115,8 @@ void format_topics_with_type(
   const std::unordered_map<std::string, uint64_t> & messages_size,
   bool verbose,
   std::stringstream & info_stream,
-  int indentation_spaces)
+  int indentation_spaces,
+  std::string sort_method = "name")
```

Comment:
as you suggested, this option would be nice to configure via CLI so that user can choose the sorting method as they like. and test would be ideal